general/1_terminal/Oleg-Levitskiy/task_3.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+todays_date=$(date +"%y-%m-%d")
+day_week=$(date -d $todays_date +%A)
+day_month=$(date +%d)
+weekend='Saturday Sunday'
+
+if grep -q "$day_week" <<< "$weekend"
+then
+  echo Looks like $day_month is a weekend
+else
+  echo Looks like $day_month is a working day
+fi
```

Comment:
Let's make it so it works for our colleagues from Mexico and India.

Not everyone uses English as their terminal language. There exists a more portable solution.